lib/kitchen/configurable.rb

```diff
@@ -133,6 +133,16 @@ def diagnose_plugin
       result
     end
 
+    # Returns a Hash of useful information for telemetry
+    #
+    # @return [Hash] a hash of interesting stuff
+    def telemetry
+      {
+        plugin_name: name,
+        plugin_version: self.class.version,
```

Comment:
Might need a different version lookup here. I notice during a dokken run that this comes up nil/null.